APE0.rst

```diff
@@ -0,0 +1,360 @@
+Coordination Committee members via a single vote.
+
+Such a vote is triggered when a Voting Member calls for one publicly on an
+appropriate Project-wide communication channel, and two other active Voting
+Members second the proposal.
+
+For Coordination Committee Member(s) to be removed, two-thirds of the active
+Voting Members must vote in support of the removal.
+
+To avoid repeated removal votes, any individual Voting Member who has called
+for, or seconded such a vote, may not call for or second a vote to remove any
+other Coordination Committee member for one year from the original motion.
+
+The Ombudsperson
+----------------
+The Ombudsperson represents the interests of the Astropy community by providing
+an alternative point of contact for sensitive issues such as code of conduct
+violations and ethical concerns. Candidates for this project role are publicly
+nominated by the Coordination Committee, after which the Coordination Committee
+allows at least two weeks for comment, and then the nominee must be confirmed by
+two-thirds of the active Voting Members.
+
+The Ombudsperson has no term limit but can resign at any time, or be removed by
+the same process as being confirmed: the Coordination Committee initiates, there
+is a two-week comment period, and two-thirds of the active Voting Members have
+to approve the removal. In the period between removal/resignation and new
+appointment, the Coordination Committee will temporarily take over the
+responsibilities of the Ombudsperson, should the need arise.
+
+The Voting Members
+------------------
+
+Role
+^^^^
+The Voting Members are the group of trusted individuals who operate the Astropy
+Project on behalf of the community. They have authority over the Astropy
+Project’s technical resources, including the Astropy Project website itself, the
+Astropy GitHub organization and repositories, the issue tracker, and all Astropy
+communication channels. In practice, much of this authority is passed to the
+Coordination Committee via the voting processes described in this Charter.
+
+They also assume many roles required to achieve the Project's goals, especially
+those that require a high level of trust. Collectively, they make decisions that
+shape the future of the Project.
+
+Voting Members are expected to act as role models for the community and
+custodians of the Project, on behalf of the community and all those who rely on
+Astropy. They will act as representatives of the Astropy Project, where
+necessary, including in online discussions or at official Astropy events.
+
+Powers
+^^^^^^
+Voting Members may participate in formal votes on:
+* adding new Voting Members,
+* removing Voting Members,
+* electing Coordination Committee members,
+* removing Coordination Committee members,
+* appointment or removal of the Ombudsperson,
+* changes to this Charter,
+* other matters for which the Coordination Committee believes a vote is
+  appropriate.
```

Comment:
In the previous bullet list the first letter of each item was capitalized, and here it isn't.  Personally I prefer this style, but I'm not sure which is more correct.

Reply:
Maybe we need @lglattly to have a look... 😸

Reply:
I'm not funded to work on the current grant right now, but hopefully in the near future this will be in scope for me and I'd be happy to take a look! Thanks for the heads up -- I'll add this to my future-copy editing running list!

Reply:
Oh and to answer this question, the [Astropy Narrative Style Guide](https://docs.astropy.org/en/latest/development/style-guide.html) doesn't have a specific guideline for bullet lists at this point (I could add one in the future), but I always capitalize the first letter of each item. If this looks strange in this particular case with the commas, I would either leave the commas off and do a list without punctuation, or I would put periods at the end of each item. Either periods or no periods in a bullet list is fine, as long as you are consistent within the list, and strive for consistency within all lists in the document as a whole (taking into account that some lists may be single word items in which periods would look strange and others may be sentence-long items which call for punctuation).

Reply:
Fixed to be always capitalized